atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -99,5 +95,18 @@ def run(self):
 
         self.log.debug(
```

Comment:
can we also for consistency report that if base_from_scratch:

"base image 'scratch' unchanged" or something like that

Reply:
Ack, added